### PR TITLE
Some tiny lexicon adjustments

### PR DESCRIPTION
**Pull request by anoomolu**
Fixes the advancement header fr this time (was missing from the big commit), manaweave armour to robes in the Imagination Forest advancement and adds a page in the Alchemy section for Dripleaf shrinking.

### CODE REVIEW

File: Xplat/src/main/resources/assets/botania/lang/en_us.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "_comment3": "Untranslated entries will automatically be inherited from this file.",
   "_comment4": "Once again, don't copy from English, and THANK YOU for your hard work!",
 
-  "itemGroup.botania.botania": "Botania",
+  "itemGroup.botania": "Botania",
```
Comment (TheRealWormbo) [inline Xplat/src/main/resources/assets/botania/lang/en_us.json]: I'm pretty sure changing this property will break something. Can't remember what.

Comment (anoomolu) [reply Xplat/src/main/resources/assets/botania/lang/en_us.json]: I’ve had this change in a resource pack for a while now and haven’t noticed anything wrong, hmmm

Comment (TheRealWormbo) [reply Xplat/src/main/resources/assets/botania/lang/en_us.json]: I believe it's at least the creative tab name, and there were other changes to adjust similar usages of that property.

Comment (TheRealWormbo) [reply Xplat/src/main/resources/assets/botania/lang/en_us.json]: I assume your resource pack version of en_us.json merely shadowed matching entries from the mod's own file, which is why it still worked. Anyway, I renamed the translation key uses in code now and reverted the key to what it used to be (and still is in some other language files) like you did in this PR.

File: Xplat/src/main/resources/assets/botania/lang/en_us.json [hunk 2]
```diff
@@ -489,7 +489,7 @@
   "advancement.botania:craftingHaloCraft": "Segments Blue",
   "advancement.botania:craftingHaloCraft.desc": "Craft an Assembly Halo to facilitate crafting",
   "advancement.botania:manaweaveArmorCraft": "Imagination Forest",
-  "advancement.botania:manaweaveArmorCraft.desc": "Create Manaweave Armor to power up your Rods",
+  "advancement.botania:manaweaveArmorCraft.desc": "Create Manaweave Robes to power up your Rods",
```
Comment (TheRealWormbo) [inline Xplat/src/main/resources/assets/botania/lang/en_us.json:492]: This fix also needs to happen in `en_gb`.

Comment (anoomolu) [reply Xplat/src/main/resources/assets/botania/lang/en_us.json:492]: Good catch, thank you ^^

**Pull request merged**